src/index.tsx

```diff
@@ -182,7 +182,12 @@ export {TrackingConfig, useTrackingConfig} from './utils/analytics';
 
 export {useDocumentVisibility} from './utils/document-visibility';
 
-export {ThemeVariant, useIsInverseVariant, useThemeVariant} from './theme-variant-context';
+export {
+    ThemeVariant,
+    useIsInverseVariant,
+    useIsInverseOrOverMediaVariant,
```

Comment:
is this needed in webapp? I think I preffer not making this public yet. If we need the case in webapp we could use `useThemeVariant`, unless we find we are repeating this check in many places in webapp

Reply:
Discussed offline. Yes, I've used it in 6 places in webapp and we don't see any bad thing about exporting this